index.html

```diff
@@ -145,7 +146,7 @@ <h1 style="font-size:50px">AIXI<span style="color:#058;">js</span></h1>
 		                <label title="Number of time steps to run the simulation for">Time steps:</label>
 		                <input type="number" class="params" name="cycles" id="cycles" min="1" max="1000000" step="1">
 		            </p>
-					<input type="submit" class="pure-button pure-button-primary" id="run" value="Run" onclick="demo.run()">
+					<input type="submit" class="pure-button pure-button-primary" id="run" value="Run" onclick="demo.run_noupdate()">
```

Comment:
I've changed the API so that you pass in a 'noupdate' flag. Just use demo.run(true) for noupdates.